pkg/cvo/sync_worker.go

```diff
@@ -550,12 +558,14 @@ func (w *SyncWorker) Update(ctx context.Context, generation int64, desired confi
 		klog.V(2).Info("The sync worker has already been notified that new work is available")
 	}
 
-	return w.status.DeepCopy()
+	return w.locked.status.DeepCopy()
```

Comment:
This is a live-code change tucked into the comment-focused 0153577d3c1c48.  Possibly got squashed into the wrong commit while rebasing?  And then you revert this change in 4f7b4812a6f41?  Maybe you can leave it untouched in both commits?

Reply:
Yup, bad `fixup` in a rebase, I'll clean it up.